CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## {{Next Version}}
+
+- Add tests for expired gpg keys, and gpg keys with only emails.
+
```

Comment:
Is it ok if we put in change information for the next changelog this way?

Reply:
Yes, I do the same thing.

Reply:
excellent, thank you